actionpack/lib/action_dispatch/middleware/exception_wrapper.rb

```diff
@@ -174,6 +174,31 @@ def self.status_code_for_exception(class_name)
       Rack::Utils.status_code(@@rescue_responses[class_name])
     end
 
+    def show?(request)
+      # We're treating `nil` as "unset", and we want the default setting to be
+      # `:all`. This logic should be extracted to `env_config` and calculated
+      # once.
+      config = request.get_header("action_dispatch.show_exceptions")
+
+      # For backwards compatibility.
+      case config
+      when true
+        ActionDispatch.deprecator.warn("Setting action_dispatch.show_exceptions to true is deprecated. Set to :all instead.")
+        config = :all
+      when false
+        ActionDispatch.deprecator.warn("Setting action_dispatch.show_exceptions to false is deprecated. Set to :none instead.")
+        config = :none
+      end
+      case config
```

Comment:
Can't these be a single case, just continue adding `when` branches?

Reply:
Ah, yes I see what you mean. Done in lates.